### ISSUE TITLE
Updates Needed to Edit Locality View Edit History

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#4400]
View Edit History in the Edit Locality page is confusing. I just made an edit to a locality name by correcting a mispelling in specific locality from "Reseva" to "Reserva".  But when I look at the View Edit History, my name and the date of correction show up in the same row as the original mispelling, instead of with the correction I just made. 
Suggest that the agent name and date stamp of the correction be displayed in the same row as the corrected locality. The original incorrect information show be on the previous row.  
Why this is important:
The current display fails to attribute changes made to the correct agent and date. As a user looking at this display, I would assume it means that I assigned the original "Reseva" mispelling - and it is unclear that I made the correction.

**Comment by Jegelewicz** [comment]
I agree with Mariel - it looks like she assigned the original rather than the correction. Perhaps this is "just UI"

> there's an extensive explanation at the top.

I think it has been well established that we aren't reading those and that we need to make these things as intuitive as possible. In reality, Mariel made both of these edits, no? Shouldn't she be associated both with the "deletion" in the red boxes and the "addition" of whatever replaced that information in the "currentData"?

**Comment by mkoo** [comment]
I use this all the time actually so I've gotten used to it but I appreciate that a more intuitive interface is needed. Adding to the UI project but this is low priority for now
